include/avif/avif.h

```diff
@@ -1091,6 +1091,11 @@ typedef struct avifScalingMode
 //   * NOTE: Please see the "Understanding maxThreads" comment block above
 // * Quality range: [AVIF_QUALITY_WORST - AVIF_QUALITY_BEST]
 // * Quantizer range: [AVIF_QUANTIZER_BEST_QUALITY - AVIF_QUANTIZER_WORST_QUALITY]
+// * If targetSize is set to a number of bytes strictly greater than 0, multiple values of quality and
+//   qualityAlpha are tested to find the settings leading to an encoded image with a size as close to
+//   targetSize as possible.
+//   Only modifies quality if it was not set. Only modifies qualityAlpha if it was not set.
+//   This feature is only available with AVIF_ADD_IMAGE_FLAG_SINGLE and is 7 times slower.
```

Comment:
I suggest moving this list item after the next list item (which is about the deprecated quantizer fields), and also moving the `targetSize` field after the `maxQuantizerAlpha` field  below. The reason is that the quality and quantizer fields are closely related, so it would be good to describe them with nothing else in between.